test/e2e/e2e_test.go

```diff
@@ -505,15 +505,15 @@ func TestListResults(t *testing.T) {
 
 		want, err := gc.ListResults(ctx, &resultsv1alpha2.ListResultsRequest{
 			Parent:  "default",
-			OrderBy: "created_time",
+			OrderBy: "create_time",
```

Comment:
Do we need to note this as a "breaking" change? I'm fine as long as we have an appropriate "ACTION REQUIRED" release note.

Reply:
Yep, absolutely. Since create_time and update_ttime are deprecated in the proto schema, I decided to change the fields in the order  by clause as well to simplify the API from the user perspective. Then, people do not need to perform a mental mapping between the wire representation and thee database model; they may use the same wire fields to query the API. I'll add to the release notes.